peer.go

```diff
@@ -851,10 +851,10 @@ func (p *peer) handleLocalClose(req *closeLinkReq) {
 	// out this channel on-chian, so we execute the cooperative channel
 	// closure workflow.
 	case CloseRegular:
-		closingTxid, err = p.executeCooperativeClose(channel)
-		peerLog.Infof("Attempting cooperative close of "+
-			"ChannelPoint(%v) with txid: %v", req.chanPoint,
-			closingTxid)
+		err := p.sendShutdown(channel)
+		if err != nil {
+			req.err <- err
```

Comment:
It's not strictly necessary, but in order to possibly eliminate some future bugs that arise by inserting further statements below this if, a `return` should be inserted here.